content/hi/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: क्लाउड नेटिव शब्दावली
+---
+
+# क्लाउड नेटिव शब्दावली
+
+क्लाउड नेटिव शब्दावली(Cloud Native Glossary), CNCF बिजनेस वैल्यू उपसमिति(BVS, Business Value Subcommittee) के नेतृत्व में एक परियोजना है। इसका लक्ष्य बिना किसी पूर्व तकनीकी ज्ञान की आवश्यकता के स्पष्ट और सरल भाषा में क्लाउड नेटिव अवधारणाओं की व्याख्या करना है।
+
+## योगदान देना
+
+क्लाउड नेटिव शब्दावली में परिवर्तन, परिवर्धन और सुधार का सुझाव देने के लिए सभी आमंत्रित हैं। हम इस साझा शब्दकोष को विकसित करने और इसमें सुधार करने के लिए CNCF के आधीन एक समुदाय-संचालित प्रक्रिया(lexicon) का उपयोग करते हैं। यह शब्दावली क्लाउड नेटिव(Cloud Native) तकनीकों के आसपास साझा शब्दावली को व्यवस्थित करने के लिए एक विक्रेता-तटस्थ(vendor-neutral) मंच प्रदान करती है। परियोजना के उद्देश्य और चार्टर का पालन करने वाले सभी प्रतिभागियों के योगदान का स्वागत है।
+
+जो कोई भी योगदान देना चाहता है, वह GitHub मुद्दा(issue) सबमिट कर सकता है या पुल रिक्वेस्ट(pull request) बना सकता है। कृपया सुनिश्चित करें कि आप [स्टाइल गाइड](/hi/style-guide/) का पालन करें, [कैसे योगदान दें](/hi/contribute/) आलेख पढ़ें, और CNCF Slack पर #glossary-localization-hindi चैनल से जुड़ें। जो शब्दावली का अपनी मूल भाषा में अनुवाद करने में मदद करना चाहते हैं, उनके लिए #glossary-localizations चैनल भी है।
+
+## स्वीकृतियाँ
+
+क्लाउड नेटिव शब्दावली की शुरुआत CNCF मार्केटिंग समिति(CNCF Marketing Committee, BVS) द्वारा की गई थी और इसमें [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/),
+[Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/), [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), [Seokho Son](https://www.linkedin.com/in/seokho-son/) और कई अन्य योगदानकर्ताओं के योगदान शामिल हैं।
+
```

Comment:
```suggestion

Localization of the Cloud Native Glossary was initiated by [Hindi localization team](https://cloud-native.slack.com/archives/C02PCHEQXK6) and contributed by contributions from [contributor1](https://www.linkedin.com/in/... or another link), ..., [contributorN](https://www.linkedin.com/in/seokho-son/), and many more contributors.
```

@Garima-Negi @anubha-v-ardhan 

Please add acknowledgement for major contributors for the Hindi localization :)

I think this update can be done in another PR.

Reply:
PR for ack of hindi localization team : [https://github.com/cncf/glossary/pull/664/](https://github.com/cncf/glossary/pull/664/)

Reply:
merged!